.github/workflows/biome.yml

```diff
@@ -0,0 +1,15 @@
+name: Format and lint with Biome
+
+on:
+  push:
+
+jobs:
+  format-and-lint-with-biome:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+      - name: Run Biome
+        run: biome ci .
```

Comment:
No need for version specification
> The action will look for the version of the @biomejs/biome dependency in the lockfiles of popular package managers such as npm, yarn, pnpm, and bun.

https://github.com/marketplace/actions/setup-biome#automatic-version-detection